Anima/diffusion/mcm_estimator/animaMCMEstimatorImageFilter.hxx

```diff
@@ -754,14 +730,12 @@ MCMEstimatorImageFilter<InputPixelType, OutputPixelType>
     mcmCreator->SetModelWithStaniszComponent(m_ModelWithStaniszComponent);
     mcmCreator->SetCompartmentType(Stick);
     mcmCreator->SetNumberOfCompartments(currentNumberOfCompartments);
-    mcmCreator->SetUseFixedWeights(m_UseFixedWeights || (m_MLEstimationStrategy == VariableProjection));
-    mcmCreator->SetFreeWaterProportionFixedValue(m_FreeWaterProportionFixedValue);
-    mcmCreator->SetStationaryWaterProportionFixedValue(m_StationaryWaterProportionFixedValue);
-    mcmCreator->SetRestrictedWaterProportionFixedValue(m_RestrictedWaterProportionFixedValue);
-    mcmCreator->SetStaniszProportionFixedValue(m_StaniszProportionFixedValue);
+    mcmCreator->SetUseFixedWeights(m_MLEstimationStrategy == VariableProjection);
```

Comment:
So in the end we can still set the weights to fixed values and you do that only with VARPRO? Isn't it wrong?

Reply:
No you can't, the option has disappeared from the program. I keep fixed weights for practical purposes in VARPRO (much more transparent to set the model to fixed weights to get the right parameters vectors). But I could rename the option to SetVarProMode or something like this if you wish